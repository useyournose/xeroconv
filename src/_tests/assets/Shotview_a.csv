Marlin,.357, 15,3grs W296, 158,0 gr
﻿#,Tempo (MPS),Δ DURCHSCHNITT (MPS),KE (J),Leistungsfaktor (N⋅s),Zeit,Ölschuss,Erstschuss,Schussnotizen
1,"488,3","14,2","1220,7","5,0",17:09:22,,,
2,"425,5","-48,7","926,7","4,4",17:09:27,,,
3,"477,0","2,9","1165,0","4,9",17:09:30,,,
4,"485,1","11,0","1204,9","5,0",17:09:34,,,
5,"476,8","2,7","1164,0","4,9",17:09:37,,,
6,"474,4","0,3","1152,1","4,9",17:10:04,,,
7,"470,7","-3,5","1134,0","4,8",17:10:08,,,
8,"474,8","0,7","1154,1","4,9",17:10:10,,,
9,"482,7","8,5","1192,7","4,9",17:10:13,,,
10,"486,0","11,9","1209,3","5,0",17:10:17,,,
-,,,,,,
AVERAGE SPEED,"474,1",,,,,
AVERAGE POWER FACTOR,"4,9",,,,,
STANDARDABWEICHUNG,"17,1",,,,,
STREUUNG,"62,8",,,,,
Gewicht des Projektils (GRAN),"158.0",,,,,
Trainingsnotiz,,,,,,
-,,,,,,
Datum,"JULI 02, 2024 17:08",,,,,
Alle Schüsse sind in den Berechnungen enthalten,,,,,,
