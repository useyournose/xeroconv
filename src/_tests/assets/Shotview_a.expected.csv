sep=;
Device ID;useyournose-xeroconv;;

Series No;0001;;
Total number of shots;0010;;

Units velocity;m/s;;
Units distances;m;;
Units kinetic energy;j;;
Units weight;grains (grs);;

Stats - Average;474.13;m/s;
Stats - Highest;488.3;m/s;
Stats - Lowest;425.5;m/s;
Stats - Ext. Spread;62.8;m/s;
Stats - Std. Dev;17.1;m/s;

Shot ID;V0;Ke0;Proj. Weight;Date;Time
0001;488.3;1220.7;5;158;02-07-2024;17:09:22;
0002;425.5;926.7;4.4;158;02-07-2024;17:09:27;
0003;477;1165;4.9;158;02-07-2024;17:09:30;
0004;485.1;1204.9;5;158;02-07-2024;17:09:34;
0005;476.8;1164;4.9;158;02-07-2024;17:09:37;
0006;474.4;1152.1;4.9;158;02-07-2024;17:10:04;
0007;470.7;1134;4.8;158;02-07-2024;17:10:08;
0008;474.8;1154.1;4.9;158;02-07-2024;17:10:10;
0009;482.7;1192.7;4.9;158;02-07-2024;17:10:13;
0010;486;1209.3;5;158;02-07-2024;17:10:17;
